special-interest-groups/sig-dashboard/roles-and-organization-management.md

```diff
@@ -0,0 +1,75 @@
+# Dashboard SIG Roles and Organization Management
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
+It lists common requirements, responsibilities and privileges of each role in developer group.
+
+### Join
+
+There is no restrictions to join the discussion of Dashboard SIG. See [README](./README.md) for where we discuss.
+
+However you will be only listed in our [member list] if you have **Active Contributor** role or above.
+
+### Promotion
+
+#### Concept: PR Point
+
+Most of the promotion rules below use "PR Point" to calculate the contribution, which is calculated as:
+
+- 1 easy level task = 1 point
+- 1 medium level task = 5 point
+- 1 hard level task = 15 point
+
+#### Be Promoted to [`Contributor`](../../architecture/README.md#contributor)
+
+Everyone that successfully contributes commits to the [TiDB Dashboard repository](https://github.com/pingcap-incubator/tidb-dashboard) is a contributor.
+
+#### Be Promoted to [`Active Contributor`](../../architecture/README.md#active-contributor)
+
+Please open a Pull Request to list yourself in our [member list] if you meet the following requirements and want to have an Active Contributor role:
+
+- PR point >= 5 in recent 1 years.
```

Comment:
It should be 8 PRs within one year.

Reply:
Looks like the gap between Active Contributor and Reviewer will be very small.